### PR TITLE
WIP, ENH: Adds `queryatlas` command for coordinates of interest

### LINKED ISSUE #37: Add ability to query atlases with coordinate
It'd be nice to have some basic CLI or even just easily-exposed python functionality to query the available atlases for a given coordinate. Right now users are required to provide a statistical map, but it might be nice just to say "Hey, I wonder what anatomical region coordinate [x, y, z] is in the Harvard Oxford atlas" without opening up e.g., FSL or AFNI or whatnot.

This shouldn't take too much work, since `read_atlas_cluster()` basically already does this!

### PR DESCRIPTION
**Pull request by rmarkello**
Closes #37.

Adds ability to query atlases with a single xyz coordinate (in MNI space) and get a slightly-formatted output of the labels associated with that coordinate for all available (or requested) atlases.

Example usage:
```bash
$ queryatlas 10.3 -20.1 34.8
Atlas                     Label                    
==========                ==========               
aal                       Cingulum_Mid_R
desikan_killiany          Right-Cerebral-White-Matter
destrieux                 Right-Cerebral-White-Matter
harvard_oxford             0% no_label
juelich                   73% WM_Cingulum_R
                          31% WM_Callosal_body
neuromorphometrics        Right_Cerebral_White_Matter
```

Still need to add tests, but wanted to open a PR to see if you have feedback on the idea / implementation / organization.

### CODE REVIEW

File: atlasreader/cli.py [hunk 1]
```diff
@@ -89,7 +90,51 @@ def main():
...
+    the command line!
+    """
+
+    opts = _queryatlas_parser()
+    print('{0:<25} {1:<25}\n{2:<25} {2:<25}'.format('Atlas', 'Label', '='*10))
+    for atlas in check_atlases(opts.atlas):
+        label = read_atlas_peak(atlas, opts.coordinate, opts.prob_thresh)
+        if isinstance(label, list):
+            label = '\n{}'.format(' '*26).join(
+                ['{:>2}% {}'.format(*e) for e in label]
```
Comment (miykael) [inline atlasreader/cli.py]: Very nice! crazy compact :+1:

**Comment by miykael** [comment]
@rmarkello - That's really cool! Like this, a user doesn't need to have a `stat_map` but can directly investigate coordinates. That will help me a lot because usually, I have to open `fslview` load some atlases etc.

This will directly expand the use case for `atlasreader`.

**Comment by danjgale** [comment]
This is a great enhancement! I find myself doing this all the time as well.

**Comment by miykael** [comment]
As these are only codecov failures, should we merge nonetheless?

**Comment by miykael** [comment]
Ah, and we should probably add this functionality also to the demo notebook.

**Comment by miykael** [comment]
This looks all good. I will merge this.

**Pull request merged**